src/services/codefixes/convertTypedefToType.ts

```diff
@@ -150,29 +180,28 @@ function createSignatureFromTypeLiteral(typeLiteral: JSDocTypeLiteral): Property
 
         if (typeReference && name) {
             const questionToken = isOptional ? factory.createToken(SyntaxKind.QuestionToken) : undefined;
-            const prop = factory.createPropertySignature(
+
+            return factory.createPropertySignature(
                 /*modifiers*/ undefined,
                 name,
                 questionToken,
                 typeReference
             );
-
-            return prop;
         }
     };
 
-    const props = mapDefined(propertyTags, getSignature);
-    return props;
+    return mapDefined(propertyTags, getSignature);
 }
 
 function getPropertyName(tag: JSDocPropertyLikeTag): string | undefined {
     return tag.name.kind === SyntaxKind.Identifier ? tag.name.text : tag.name.right.text;
 }
 
 /** @internal */
-export function getJSDocTypedefNode(node: Node): JSDocTypedefTag | undefined {
+export function getJSDocTypedefNodes(node: Node): readonly JSDocTag[] {
     if (hasJSDocNodes(node)) {
-        return forEach(node.jsDoc, (node) => node.tags?.find(isJSDocTypedefTag));
+        return flatMapToMutable(node.jsDoc, (doc) => doc.tags?.filter((tag) => isJSDocTypedefTag(tag)));

```

Comment:
Could this just be `flatMap`? There's only one use and it doesn't actually need a copy, so `flatMap` should be more efficient.